shared/src/main/scala/drt/shared/PcpUtils.scala

```diff
@@ -13,4 +13,7 @@ object PcpPax {
       case (Some(apiPax), _, _, _) => apiPax
       case _ => defaultPax
     }
+
+  def walkTime(arrival: Arrival, timeToChox: Long, firstPaxOff: Long): Option[Long] =
+    arrival.PcpTime.map(pcpTime => pcpTime - (arrival.bestArrivalTime(timeToChox) + firstPaxOff))
```

Comment:
just question here ,If we don't have pcptime then we dont display walktime ? I guess we get it as from live feeds?